core/src/main/java/org/jruby/truffle/nodes/core/BignumNodes.java

```diff
@@ -192,7 +192,7 @@ public RubyBignum pow(RubyBignum a, RubyBignum b) {
 
     }
 
-    @CoreMethod(names = "/", required = 1)
+    @CoreMethod(names = {"/", "__slash__"}, required = 1)
```

Comment:
Rubinius calls it from here https://github.com/jruby/jruby/pull/2414/files#diff-1eabf0e4bde8dc8dce596aa4d27d7042R41. I didn't do the digging yet to figure out if it's supposed to mean something special - such as never monkey patched.